lib/tasks/solid_queue_monitor.rake

```diff
@@ -0,0 +1,61 @@
+namespace :solid_queue do
+  desc "Monitor and clean up stuck SolidQueue jobs"
+  task monitor: :environment do
+    puts "=== SolidQueue Health Check ==="
```

Comment:
As far as I understand, this task will be run by cron.
Where will we be able to see these results?
Also, since we already have some admin tools for jobs. Don't they do the same thing?

Reply:
sorry. this one wasn't supposed to go into the PR

Reply:
We could leave it to use manually if required.

Reply:
`rails solid_queue:cleanup`